### DISCUSSION TITLE
HOME ASSISTANT : Reutiliser les données ailleurs que dans le menu Energie ?

### DISCUSSION THREAD

**Discussion by adrienc45** [bokub/conso-api#58] [Idées]
Bonjour.

Grace à l'excellent tutoriel livré avec le module , l’intégration dans Home Assistant s'est bien passé.
Mon Linky est visible dans "Appareils" et "Entités".

Je m'attendais à ce que les données de consommation journalières soient visibles dans la section "Capteurs" de mon " Appareil", ce n'est apparemment pas prévu.

Voici ma question : Quelqu'un de la communauté pourrait-il me dire ou sont stockées les données journalières récupérées par le module et donc visible grâce au tuto dans mon menu "Énergie" ? 

Je souhaite les recuperer et créer dans mon HomeAssistant des variables statistiques ( conso mensuelle , annuelle , augmentation / diminutions anormales de ma conso en comparaison avec les jours précédents , ce ne sont que quelques exemples parmi pleins d'autres possibilités .. )

![Capture d’écran 2024-07-28 141455](https://github.com/user-attachments/assets/4ef28dcc-ac2e-4f8a-8a38-bc6b3c7f0bed)

Merci d'avance.

**Comment by bokub** [comment]
Salut @adrienc45

Tu auras plus de chances d'obtenir des réponses en postant dans le bon repository (https://github.com/bokub/ha-linky), ou sur le forum HACF (https://forum.hacf.fr/t/add-on-pour-compteurs-linky/27608).
 
 D'ailleurs, je pense que tu trouveras peut-être des réponses dans les issues existantes de HA Linky: https://github.com/bokub/ha-linky/issues
